docs/Model_Assumptions.rst

```diff
@@ -102,10 +106,42 @@ For two peak demand pricing periods, the resulting dispatch could look as follow
 Energy storage
 ##############
 
-Generic storages are defined with file `energyStorage.csv` and `storage_*.csv` and have subassets, which are listed in :ref:`storage_csv`.
+Energy storages such as battery storages, thermal storages or H2 storages are modelled with the *GenericStorage* component of *oemof.solph*. They are designed for one input and one output and are defined with files `energyStorage.csv` and `storage_*.csv` and have several parameters, which are listed in the section :ref:`storage_csv`.
+
+The state of charge of a storage at the first and last time step of an optimization are equal.
+Charge and discharge of the whole capacity of the energy storage are possible within one time step in case the capacity of the storage is not optimized. In case of
+capacity optimization charge and discharge is limited by the :ref:`crate-label`.
+
+.. _battery_storage:
+
+Battery energy storage system (BESS)
+====================================
+
+BESS are modelled as *GenericStorage* like described above. The BESS can either be connected directly to the electricity bus of the LES or via a charge controller that manages the BESS.
+When choosing the second option, the capacity of the charge controller can be optimized, which takes its specific costs and lifetime into consideration.
+If you do not want to optimize the charge controller's capacity you can take its costs and efficiency into account when defining the storage's input and output power, see :ref:`storage_csv`.
+A charge controller is defined by two transformers, see section :ref:`energy_conversion` above.
+
+Note that capacity reduction over the lifetime of a BESS that may occur due to different effects during aging cannot be taken into consideration in MVS. A possible workaround for this could be to work with the c-rate of the storage or the efficiencies of input and output flows. Note that defining the c-rate as a time series has not been tested, yet, see section :ref:`time_series_params`.
```

Comment:
The change in capacity also happens during periods < 1 year, as I remember, there are reversible and irreversible effects. The reduction gets higher during the lifetime. 
So, for effects during period < 1 year you would need to work with time series. However, this is a problem:
> then this would imply that it was possible to jump back to the initial values in the 2nd year of operation.

Will change that in next commit.